applications/MeshingApplication/custom_processes/metrics_hessian_process.h

```diff
@@ -88,26 +89,31 @@ class ComputeHessianSolMetricProcess
     ComputeHessianSolMetricProcess(
         ModelPart& rThisModelPart,
         TVarType& rVariable,
-        const double rMinSize,
-        const double rMaxSize,
-        const bool rEnforceCurrent = true,
-        const double rInterpError = 1.0e-6,
-        const double rMeshConstant = 0.28125, // TODO: This is for tetrahedron, look in literature for more
-        const double rAnisRatio = 1.0,
-        const double rBoundLayer =  1.0,
-        const std::string rInterpolation = "Linear"
+        Parameters ThisParameters = Parameters("{'minimal_size': 0.1,'maximal_size': 10.0, 'enforce_current': true, 'hessian_strategy_parameters': { 'interpolation_error': 1.0e-6, 'mesh_dependent_constant': 0.28125}, 'anisotropy_remeshing': true, 'anisotropy_parameters': {'hmin_over_hmax_anisotropic_ratio': 1.0, 'boundary_layer_max_distance': 1.0, 'interpolation': 'Linear'}}")
```

Comment:
do not pass this here.

define it inside the body and use the validation mechanism!